CHANGELOG.md

```diff
@@ -5,12 +5,13 @@
 * Added Appear and dismiss callback for nps/survey widgets
 * Added "recordIndirectAttribution" method
 * Added "recordDirectAttribution" method (Currently supporterd only for Android)
+* Added "setUserLocation" method to set user location
```

Comment:
Mention that the old call was deprecated

Reply:
and also mention the old initLocation call

Reply:
"Deprecated old init config methods. You should use the config object now. Those methods are: ..."